AnnoyancesFilter/sections/popups.txt

```diff
@@ -99,6 +99,7 @@ sodapdf.com##.be-tooltip
 !
 indirimdemarket.com#$##social-media-modal { display: none!important; }
 indirimdemarket.com#$#body { overflow: visible!important; }
+~connect.deezer.com,indirimdemarket.com,sk7mobile.com,calkalasutasayrani.com,egyptianstreets.com,aipa551.com,aipa552.com,baba-mail.co.il,catwalkyourself.com,bkv.hu,tfc.tv,diziweb.net,blogkurdu.net,bitfeed.co,ports.com.ua,opensnow.com,watch5s.rs,deezer.com,pr-cy.ru,bitdefender.com##.modal-backdrop
```

Comment:
Please, check out the sites that can be merged with `##.modal-backdrop`. Maybe some of them are obsolete and you can remove them from the filters now. Also if you found that the website have problem with page scrolling, don't add it to the `##.modal-backdrop`.

This one can't be in this group, because it's leftover from the popup that we don't block on cb platforms.
You should remove `indirimdemarket.com` from this list and make a `indirimdemarket.com#$#.modal-backdrop { display: none !important; }`, then move it next to the rules. Note, we have them https://github.com/AdguardTeam/AdguardFilters/commit/03b439800f459de30dbe0164ad2db29a42a28191 in `Annoyances` and `Social` filters.
```
indirimdemarket.com#$##social-media-modal { display: none!important; }
indirimdemarket.com#$#body { overflow: visible!important; }
```

Reply:
Where should I put the rule?

Reply:
`indirimdemarket.com#$#.modal-backdrop { display: none !important; }` here. The same for AdGuard Social filter.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/109816238-9174c700-7c41-11eb-96fb-a77838843a57.png)

</details><br/>

Reply:
What should add for AdGuard Social Media Filter? I don't any element.

Reply:
```
indirimdemarket.com#$##social-media-modal { display: none!important; }
indirimdemarket.com#$#body { overflow: visible!important; }
```
Those rules are in the AdGuard Social - `specific.txt`, but should be in the `popups.txt`. Move them with `indirimdemarket.com#$#.modal-backdrop { display: none !important; }` to the AdGuard Social - `popups.txt`.

Reply:
Please check now is it correct?